app/src/main/java/de/danoeh/antennapod/fragment/CategoriesFragment.java

```diff
@@ -20,7 +20,7 @@
 
     public static final String TAG = "CategoriesFragment";
     private ArrayAdapter listAdapter;
-    private GridView gridView;
+    private ListView gridView;
```

Comment:
You changed to ListView but the name of the variable is not representative now since its named gridView.